### PR TITLE
Ewas proof read

### PR DESCRIPTION
**Pull request by kpoterlowicz**

### CODE REVIEW

File: topics/epigenetics/tutorials/ewas-suite/tutorial.md [hunk 1]
```diff
@@ -108,16 +108,16 @@ The first step of EWAS data anylalysis is the raw methylation data loading (inte
...
+> 3.  Once you have inspected the quality control run **minfi rset** {% icon tool %} on the  'MethylSet'output from minfi_read450K  to convert methylation data from the `MethylSet` to a ratio of the methylated probe intensity and the overall intensity (sum of methylated and unmethylated probe intensities)
 > 4. Run **minfi maptogenome** {% icon tool %} on the output from the previous step to map ratio data to the genomic positions
 >
 > > ### {% icon comment %} Preprocess and Normalize data
-> > If your files require normalisation, you might prefer to use other of preprocessing tools provided in EWAS suite i.e. **minfi ppfun** {% icon tool %} or **minfi ppquantile**  {% icon tool %} look for recommendation at (ref).
+> > If your files require normalisation, you might prefer to use on fo the other preprocessing tools provided in EWAS suite i.e. **minfi ppfun** {% icon tool %} or **minfi ppquantile**  {% icon tool %} look for recommendation at (ref).
```
Comment (bgruening) [inline topics/epigenetics/tutorials/ewas-suite/tutorial.md]: this seems wrong?

File: topics/epigenetics/tutorials/ewas-suite/tutorial.md [hunk 2]
```diff
@@ -52,7 +52,7 @@ GSM1588707 | resistant   | BRAFi
 
 # 450K Array Data Upload
 
-The first step of EWAS data anylalysis is the raw methylation data loading (intensity information files for each two color micro array)
+The first step of EWAS data anyalysis is the raw methylation data loading (intensity information files for each two color micro array)
```
Comment (shiltemann) [inline topics/epigenetics/tutorials/ewas-suite/tutorial.md]:
```suggestion
The first step of EWAS data analysis is the raw methylation data loading (intensity information files for each two color micro array)
```

File: topics/epigenetics/tutorials/ewas-suite/tutorial.md (hunk 1 above)
Comment (kpoterlowicz) [reply topics/epigenetics/tutorials/ewas-suite/tutorial.md]:
```suggestion
> > If your files require normalisation, you might prefer to use one of the other preprocessing tools provided in EWAS suite i.e. **minfi ppfun** {% icon tool %} or **minfi ppquantile**  {% icon tool %} look for recommendation at (ref).
```

**Comment by bgruening** [comment]
Merci!

**Pull request merged**